inference-engine/src/multi_device/multi_device.cpp

```diff
@@ -74,6 +85,45 @@ MultiDeviceAsyncInferRequest::MultiDeviceAsyncInferRequest(
     _multiDeviceExecutableNetwork{multiDeviceExecutableNetwork},
     _inferRequest{inferRequest},
     _needPerfCounters{needPerfCounters} {
+    _pipeline.clear();
+    struct CheckRemoteBlobs : public ITaskExecutor {
+        explicit CheckRemoteBlobs(MultiDeviceInferRequest* _request_,
+                                  MultiDeviceExecutableNetwork* network)
+                : _request{_request_}, _network(network) {}
+        void run(Task task) override {
```

Comment:
I think here you can use `InferenceEngine::ImmediateExecutor` instead of `CheckRemoteBlobs`. And move `run()` code to the pipline stage task

Reply:
good point, fixed